core/controllers/collection_viewer.py

```diff
@@ -93,12 +93,16 @@ def get(self, collection_id):
                     allow_invalid_explorations=False))
         except Exception as e:
             raise self.PageNotFoundException(e)
+        collection_summary = \
+        summary_services.get_displayable_collection_summary_dicts_matching_ids(
+            collection_id)
 
         self.values.update({
             'can_edit': (
                 self.user_id and rights_manager.Actor(self.user_id).can_edit(
                     feconf.ACTIVITY_TYPE_COLLECTION, collection_id)),
             'collection': collection_dict,
+            'collection_summary': collection_summary,
```

Comment:
What info do you need from 'collection_summary' that isn't already in 'collection'?